.gitignore

```diff
@@ -5,6 +5,10 @@
 /.pnp
 .pnp.js
 
+
```

Comment:
Any reason why you are ignoring these files?

Reply:
Yeah I was to ask you for permission to delete it.
They are no longer needed.

Reply:
You can delete them if they are no longer needed